docs/resources/application_user.md

```diff
@@ -26,7 +26,7 @@ resource "humanitec_application_user" "another_owner" {
 ### Required
 
 - `app_id` (String) The Application ID.
-- `role` (String) The role that this user holds
+- `role` (String) The role that this user holds. Could be `viewer`, `developer` or `owner`.
```

Comment:
The docs are auto-generated from markdown descriptions inside the code, could you update https://github.com/humanitec/terraform-provider-humanitec/blob/main/internal/provider/resource_application_user.go#L82 (and the other places you touched) and run `go generate ./...` afterwards?